phpstan.neon

```diff
@@ -12,6 +11,8 @@ parameters:
         - 'tests/Application/src/**.php'
 
     ignoreErrors:
+        -
+            identifier: missingType.iterableValue
```

Comment:
Can you provide an explanation as it's done below?

Reply:
It is just to mitigate an information about deprecated value:
`checkMissingIterableValueType: false`
this identifier does a same thing :)